styles.css

```diff
@@ -486,3 +496,35 @@ textarea {
     margin: 0 auto;
   }
 }
+
+/* Modal */
+
+.modal {
+  background-color: rgba(96, 112, 255, 0.9);
+  height: 100vh;
+  position: absolute;
+  z-index: 4;
+  width: 100%;
+}
```

Comment:
- [ ] Hi @DavidEspinoG, having made a great effort working on the mobile menu aspect of your portfolio project, I noticed there are still some things you need to make changes on. According to what we have in the Figma design after the mobile menu is displayed the content underneath it is `blur` but in your own design the contents are not `blur`.
You can achieve blurring of the background content by make use of the CSS property called `backdrop-filter: blur(5px);`

Figma design 👇 

![real-mob](https://user-images.githubusercontent.com/78634841/215605336-f4163c93-03b0-4f1d-9c38-571e5fb6c88b.png)

Your design 👇 

![dav](https://user-images.githubusercontent.com/78634841/215605496-87f7b922-0b81-4d51-a35a-3d473c6c6875.png)